### ISSUE TITLE
Accordion: Adjust Contrast ratio of open Accordion headers - 1.4.3 Contrast (Minimum) 

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#54]
On the open accordion headers, the text is white on gray when hovered, causing a insufficient contrast ratio of 3.54:1 (all text contrast should be minimum 4.5:1)

**Comment by syarwick23** [comment]
Color is now #988F72 with white text.(3.22:1)

**Comment by stephenravenscraft** [comment]
Updated background color to #594F40 on :hover. New contrast ratio is 8.02:1. PASS
